app/controllers/courses.js

```diff
@@ -100,7 +100,7 @@ export default Controller.extend({
   selectedYear: computed('model.years.[]', 'yearTitle', function() {
     const years = this.get('model.years');
     if(isPresent(this.yearTitle)){
-      return years.find(year => year.get('title') === parseInt(this.yearTitle, 10));
+      return years.find(year => year.get('id') === this.yearTitle);
```

Comment:
Soon we'll have linting for it, but for now can you make this `year.id` and remove the `get`?

Reply:
:heavy_check_mark: